packages/gestalt/src/textTypes.js

```diff
@@ -23,6 +23,15 @@ export const allowedColors = [
   'red',
   'watermelon',
   'white',
+  'default',
+  'subtle',
+  'success',
+  'error',
+  'warning',
+  'inverse',
+  'shopping',
+  'light',
+  'dark',
 ];
 
 export type Color =
```

Comment:
are we using this export type Color anywhere? maybe we could remove it. With autogenerated prop tables, we are inlining. could be worth checking

Reply:
I'll look into this when we do this for Heading